### PR TITLE
Reinsert fix for #5456

### PR DESCRIPTION
**Pull request by adferrand**
Dependencies generated by the script introduced with #6839 were not including anymore the fix about `enum34` for CentOS 6.

This PR reinserts this fix, and updates the script overrides to ensure that this fix will stay in next dependencies generation.

### CODE REVIEW

File: letsencrypt-auto-source/rebuild_dependencies.py [hunk 1]
```diff
@@ -42,6 +42,9 @@
     # Too touchy to move to a new version. And will be removed soon
     # in favor of pure python parser for Apache.
     'python-augeas': '0.5.0',
+    # Package enum34 needs to be explicitly limited to Python2.x, in order to avoid
+    # certbot-auto to fail on Centos 6 + Python 3.6, see #5456.
```
Comment (bmw) [inline letsencrypt-auto-source/rebuild_dependencies.py]:
```suggestion
    # certbot-auto failures on Python 3.6+ which enum34 doesn't support. See #5456.
```

**Comment by bmw** [comment]
codecov was being silly but I wanted to get this merged so I went ahead and overrode it.

**Comment by bmw** [comment]
Ah. The problem was AppVeyor failed to send coverage data to codecov. See https://ci.appveyor.com/project/certbot/certbot/builds/23552940/job/xfk2rn5ysivm95gl#L810.

**Comment by adferrand** [comment]
Spurious failure? Codecov servers seem to have closed unexpectedly the connection.

**Comment by bmw** [comment]
Yeah. I think it was just a temporary problem.

**Pull request merged**